org.btchd.btchd-qt.json

```diff
@@ -0,0 +1,88 @@
+{
+  "id": "org.btchd.btchd-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "20.08",
+  "command": "btchd-qt",
+  "finish-args": [
+    /* Xorg support */
+    "--socket=x11",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network",
+    /* Tor auth cookie */
+    "--filesystem=/run/tor/control.authcookie:ro",
+    "--filesystem=/var/run/tor/control.authcookie:ro"
```

Comment:
```suggestion
```

I think all modern distros make `/var/run` a symlink to `/run`, and it's not going to launch if that's the case.